include/nbl/ui/IInputEventChannel.h

```diff
@@ -97,21 +97,28 @@ namespace impl
 
 struct SMouseEvent
 {
+    SMouseEvent()
+    {
+        using namespace std::chrono;
+        timeStamp = duration_cast<microseconds>(system_clock::now().time_since_epoch());
```

Comment:
oh fuck no, don't do that, either get the timestamp directly from the OS, or record the time **as soon as you return from polling for the event**

Reply:
TODO: @sadiuk